_includes/header.html

```diff
@@ -15,7 +15,7 @@ <h1>
       <li><a href="{{ site.baseurl }}/contact.html">Contact</a></li>
     </ul>
     <ul class="menu external">
-      <li><a href="https://kiwiirc.com/nextclient/irc.libera.chat:+6697/?nick=Guest?#supertux">Chat</a></li>
+      <li><a href="https://kiwiirc.com/nextclient/irc.libera.chat:+6697/?nick=Guest?#supertux">IRC</a></li>
```

Comment:
```suggestion
      <li><a href="ircs://irc.libera.chat/#supertux">IRC</a></li>
```

This change requires you to have an IRC client to use the link, so no one who doesn't know what an IRC client would be able to use the button.